docs/docs/building-apps/02-app-mempool.md

```diff
@@ -51,3 +51,16 @@ Now that we have walked through the `PrepareProposal` & `ProcessProposal`, we ca
 There are countless designs that an application developer can write for a mempool, the core team opted to provide a simple implementation of a nonce mempool. The nonce mempool is a mempool that keeps transactions from an sorted by nonce in order to avoid the issues with nonces. 
 
 It works by storing the transation in a list sorted by the transaction nonce. When the proposer asks for transactions to be included in a block it randomly selects a sender and gets the first transaction in the list. It repeats this until the mempool is empty or the block is full. 
+
+
+### Configurations
+
+#### MaxTx
+Its an integer value that sets the mempool in one of three modes, bounded, unbounded, or disabled. 
+
+| MaxTx Mode | Behavior                                                               |
```

Comment:
`MaxTx` is not indicative of what is being done here. I would simply call this `mode`.

Reply:
MaxTx  (when a positive integer) is the maximum capacity of the mempool.

Reply:
I find this section confusing then.

1. I would use `MaxTxs` instead.
2. Don't use a table layout, rather bullet points intead